src/main/java/com/uber/cadence/samples/hello/HelloPeriodic.java

```diff
@@ -24,131 +25,136 @@
-        // Activities are stateless and thread safe. So a shared instance is used.
-        worker.registerActivitiesImplementations(new GreetingActivitiesImpl());
-        // Start listening to the workflow and activity task lists.
-        worker.start();
+  public static void main(String[] args) throws InterruptedException {
+    // Start a worker that hosts both workflow and activity implementation
+    Worker worker = new Worker(DOMAIN, TASK_LIST);
+    // Workflows are stateful. So need a type to create instances.
+    worker.registerWorkflowImplementationTypes(GreetingWorkflowImpl.class);
+    // Activities are stateless and thread safe. So a shared instance is used.
+    worker.registerActivitiesImplementations(new GreetingActivitiesImpl());
+    // Start listening to the workflow and activity task lists.
+    worker.start();
 
-        // Start a workflow execution. Usually it is done from another program.
-        WorkflowClient workflowClient = WorkflowClient.newInstance(DOMAIN);
-        // To ensure that this daemon type workflow is always running try to start it periodically
-        // ignoring duplicated exception.
-        // It is only to protect from application level failures.
-        // Failures of a workflow worker don't lead to workflow failures.
-        WorkflowExecution execution = null;
-        while (true) {
-            // Print reason of failure of the previous run, before restarting.
-            if (execution != null) {
-                UntypedWorkflowStub workflow = workflowClient.newUntypedWorkflowStub(execution);
-                try {
-                    workflow.getResult(Void.class); //
-                } catch (WorkflowException e) {
-                    System.out.println("Previous instance failed:\n" + Throwables.getStackTraceAsString(e));
-                }
-            }
-            // New stub instance should be created for each new workflow start.
-            GreetingWorkflow workflow = workflowClient.newWorkflowStub(GreetingWorkflow.class);
-            try {
-                execution = WorkflowClient.start(workflow::greetPeriodically,
-                        "World", Duration.ofSeconds(1));
-                System.out.println("Started " + execution);
-            } catch (DuplicateWorkflowException e) {
-                System.out.println("Still running as " + e.getExecution());
-            } catch (Throwable e) {
-                e.printStackTrace();
-                System.exit(1);
-            }
-            // This value is so low just for the sample purpose. In production workflow
-            // it is usually much higher.
-            Thread.sleep(10000);
+    // Start a workflow execution. Usually it is done from another program.
+    WorkflowClient workflowClient = WorkflowClient.newInstance(DOMAIN);
+    // To ensure that this daemon type workflow is always running try to start it periodically
+    // ignoring duplicated exception.
```

Comment:
ignoring duplicated exception -> ignoring the duplicated exception OR ignoring duplicated exceptions